scripts/superdesk/filters.js

```diff
@@ -78,6 +78,34 @@
                 return angular.isDefined(format_string) ? moment_timestamp.format(format_string) : moment_timestamp.format();
             };
         }])
+        .filter('formatRelativeDate', [function() {
+            return function(input, format_string) {
+                var COMPARE_FORMAT = 'YYYY-M-D';
+                var DISPLAY_DATE_FORMAT = 'D. MMMM YYYY [at] HH:mm';
+                var DISPLAY_CDATE_FORMAT = 'D. MMMM [at] HH:mm';
+                var DISPLAY_DAY_FORMAT = 'dddd, ';
+                var DISPLAY_TODAY_FORMAT = '[Today], ';
+                var rday, rdate;
+
+                var date = moment.utc(input);
+
+                date.local(); // switch to local time zone
```

Comment:
I don't think it should switch to local date. We need to show what ever the user sets the embargo date/time without conversion.